consensus/src/aggregation/metrics.rs

```diff
@@ -15,55 +15,57 @@ pub struct Metrics<E: RuntimeMetrics + Clock> {
     pub acks: status::Counter,
     /// Number of threshold signatures produced
     pub threshold: Counter,
-    /// Number of rebroadcast attempts by status
-    pub rebroadcast: status::Counter,
     /// Histogram of application digest durations
     pub digest_duration: histogram::Timed<E>,
 }
 
 impl<E: RuntimeMetrics + Clock> Metrics<E> {
     /// Create and return a new set of metrics, registered with the given context.
     pub fn init(context: E) -> Self {
-        let clock = Arc::new(context.clone());
```

Comment:
This is an example of a place where we clone the context provided unnecessarily. It shouldn't cause any issues per se but the context tree would look odd.